src/FieldTrait.php

```
@@ -361,15 +356,16 @@ protected function fieldFixStepArgument(string $argument): string {
    * @Given browser validation for the form :selector is disabled
    */
   public function disableFormBrowserValidation(string $selector): void {
-    $escapedSelector = addslashes($selector);
+    $selector_js = json_encode($selector, JSON_UNESCAPED_SLASHES);
```

PHPMD (2.15.0)

The variable $selector_js is not named in camelCase. (undefined)

(CamelCaseVariableName)